corehq/apps/registration/templates/registration/domain_request.html

```diff
@@ -1,8 +1,23 @@
 {% extends "registration/base.html" %}
 {% load i18n %}
+{% load hq_shared_tags %}
 
 {% block title %}New Project Space{% endblock title %}
 
+{% block js %}{{ block.super }}
+    <script src="{% static 'hqwebapp/js/lib/select2/select2.js' %}"></script>
+{% endblock %}
+
+{% block head %}{{ block.super }}
+    <link href="{% static 'hqwebapp/js/lib/select2/select2.css' %}" rel="stylesheet" />
+{% endblock %}
```

Comment:
There are multiple views that use this form (DomainRegistrationForm) and adding this at multiple views that use one single template doesn't seem like the best way.  select2 is usually associated with an input field in a Form. So may be it's better to couple the select2 snippet with a Form than with a view?